action/run-in-renode.py

```diff
@@ -540,11 +176,11 @@ def setup_python():
     Install previously downloaded python packages in emulated linux.
     """
 
-    child = px_spawn("telnet 127.0.0.1 1234", encoding="utf-8", timeout=None)
+    child = px_spawn("telnet 127.0.0.1 1234", encoding="utf-8", timeout=10)
 
     try:
-        child.expect_exact("'^]'.")
-        child.sendcontrol("c")
+        child.expect_exact("'^]'.", timeout=30)
+        child.sendline('')
```

Comment:
Why this change?

Reply:
I have added timeouts to every command, so i think there will not be such situations like 6h random tests.

Reply:
I replaced sendcontrol('c') with sendline('') because as I see this second approach is more idiomatic. And I think it is kind of rude to send interrupts when they are not needed :)